configure

```diff
@@ -6316,6 +6316,7 @@ echo_version() {
 
 if test -n "$validator_lib"; then :
    QEMU_LDFLAGS="-L$validator_lib $QEMU_LDFLAGS"
+   LIBS="$libs_validator $LIBS"
 fi
 
 if test -n "$validator_include}"; then :
```

Comment:
Should we clean-up the extra '}' that Austin identified here and in the recently copied location?

Reply:
Got it